### PR TITLE
Upgrade to ES 7.4.0 and release 7.4.0.0

### LINKED ISSUE #222: Release for ES 7.4.0.
Release notes: https://www.elastic.co/guide/en/elasticsearch/reference/7.4/release-notes-7.4.0.html

### PR DESCRIPTION
**Pull request by lukas-vlcek**
Closes #222

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 group = org.elasticsearch.plugin.prometheus
 
-version = 7.3.2.1-SNAPSHOT
+version = 7.4.0.0
```
Comment (lukas-vlcek) [inline gradle.properties]: @vvanholl Do I remember it correctly that such value of the `version` (without `-SNAPSHOT`) will trigger a new release automatically?

**Comment by lukas-vlcek** [comment]
@vvanholl PTAL

File: gradle.properties (hunk 1 above)
Comment (vvanholl) [reply gradle.properties]: YES, omitting the -SNAPSHOT suffix will make the CI do a new release while you stay on master branch.

**Comment by agaengel** [comment]
Can we expect this to be released soon ?

File: gradle.properties (hunk 1 above)
Comment (lukas-vlcek) [reply gradle.properties]: I updated the PR and included also update to README to document new release.

**Comment by lukas-vlcek** [comment]
@vvanholl PTAL. I think this should be ready to merge.

**Pull request merged**